src/libraries/System.Globalization/tests/CultureInfo/CultureInfoAll.cs

```diff
@@ -525,71 +525,482 @@ private int GetLocaleInfoAsInt(CultureInfo ci, uint lcType)
+            yield return new object[] {  0x7c67, new string[] { "ff-latn" }              , "ff-Latn-SN"    , "ful", "FUL", "ff-Latn"       , "ff-Latn-SN" };
+            yield return new object[] { 0x10407, new string[] { "de-de_phoneb", "de-de" }, "de-DE"         , "deu", "DEU", "de-DE"         , "de-DE" };
+            yield return new object[] { 0x1040e, new string[] { "hu-hu_technl", "hu-hu" }, "hu-HU"         , "hun", "HUN", "hu-HU"         , "hu-HU" };
+            yield return new object[] { 0x10437, new string[] { "ka-ge_modern", "ka-ge" }, "ka-GE"         , "kat", "KAT", "ka-GE"         , "ka-GE" };
+            yield return new object[] { 0x20804, new string[] { "zh-cn_stroke", "zh-cn" }, "zh-CN"         , "zho", "CHS", "zh-Hans-CN"    , "zh-CN" };
+            yield return new object[] { 0x21004, new string[] { "zh-sg_stroke", "zh-sg" }, "zh-SG"         , "zho", "ZHI", "zh-Hans-SG"    , "zh-SG" };
+            yield return new object[] { 0x21404, new string[] { "zh-mo_stroke", "zh-mo" }, "zh-MO"         , "zho", "ZHM", "zh-Hant-MO"    , "zh-MO" };
+            yield return new object[] { 0x30404, new string[] { "zh-tw_pronun", "zh-tw" }, "zh-TW"         , "zho", "CHT", "zh-Hant-TW"    , "zh-TW" };
+            yield return new object[] { 0x40404, new string[] { "zh-tw_radstr", "zh-tw" }, "zh-TW"         , "zho", "CHT", "zh-Hant-TW"    , "zh-TW" };
+            yield return new object[] { 0x40411, new string[] { "ja-jp_radstr", "ja-jp" }, "ja-JP"         , "jpn", "JPN", "ja-JP"         , "ja-JP" };
+            yield return new object[] { 0x40c04, new string[] { "zh-hk_radstr", "zh-hk" }, "zh-HK"         , "zho", "ZHH", "zh-Hant-HK"    , "zh-HK" };
+            yield return new object[] { 0x41404, new string[] { "zh-mo_radstr", "zh-mo" }, "zh-MO"         , "zho", "ZHM", "zh-Hant-MO"    , "zh-MO" };
+            yield return new object[] { 0x50804, new string[] { "zh-cn_phoneb", "zh-cn" }, "zh-CN"         , "zho", "CHS", "zh-Hans-CN"    , "zh-CN" };
+            yield return new object[] { 0x51004, new string[] { "zh-sg_phoneb", "zh-sg" }, "zh-SG"         , "zho", "ZHI", "zh-Hans-SG"    , "zh-SG" };
+        }                               
+                                        
+        [Theory]                        
         [MemberData(nameof(CultureInfo_TestData))]
-        public void LcidTest(string cultureName, int lcid, string specificCultureName, string threeLetterISOLanguageName, string threeLetterWindowsLanguageName, string alternativeCultureName, string consoleUICultureName)
-        {
-            _ = alternativeCultureName;
+        public void LcidTest(int lcid, string[] cultureNames, string specificCultureName, string threeLetterISOLanguageName, string threeLetterWindowsLanguageName, string alternativeCultureName, string consoleUICultureName)
+        {                               
+            _ = alternativeCultureName; 
```

Comment:
nit: trailing spaces.
(might make it easier to validate/review, if test enhancements are split in a separate PR)

Reply:
Extending the test helped pick up a couple issues in the data; e.g.

![image](https://user-images.githubusercontent.com/1142958/101303689-7eb34780-3836-11eb-8a9f-dd22948240b5.png)